rules/scrum-master-do-you-schedule-the-3-meetings/rule.md

```diff
@@ -68,8 +84,8 @@ See rule [What happens at a Sprint Retrospective Meeting?](/do-you-know-what-hap
 
 **Sprint Planning Meeting**
 
-We go through the backlog (aka to-do list), get more information, estimate and then prioritize.   
-We then breakdown to tasks and commit to what we believe we can deliver for the next Sprint.   
+We go through the backlog (aka to-do list), get more information, estimate and then prioritize.
+We then breakdown to tasks and commit to what we believe we can deliver for the next Sprint.
```

Comment:
**question**: The spaces at the end were likely intentional to cause line-breaks - are you sure you wanted to remove them?

Reply:
No - I reverted that. Thanks for catching this ;)